includes/class-freemius.php

```diff
@@ -17088,12 +17164,13 @@ private function _load_account() {
                 $this->_register_account_hooks();
             }
 
-            if (
-                $this->is_user_in_admin() &&
-                $this->is_clone() &&
-                empty( FS_Clone_Manager::instance()->get_clone_identification_timestamp() )
-            ) {
-                FS_Clone_Manager::instance()->store_clone_identification_timestamp();
+            if ( $this->is_user_in_admin() && $this->is_clone() ) {
+                if ( empty( FS_Clone_Manager::instance()->get_clone_identification_timestamp() ) ) {
+                    FS_Clone_Manager::instance()->store_clone_identification_timestamp();
+                }
+
+                FS_Clone_Manager::instance()->maybe_update_clone_resolution_support_flag( $this->_storage->sdk_last_version );
+                $this->send_pending_clone_update();

```

Comment:
@fajardoleo I'd propose to add `_once` to the end of the method's name making it clear it only happens once.